Mixpanel/MPNetwork.m

```diff
@@ -279,7 +279,7 @@ + (BOOL)parseHTTPFailure:(NSHTTPURLResponse *)response withError:(NSError *)erro
 
 - (void)updateNetworkActivityIndicator:(BOOL)enabled {
 #if !MIXPANEL_LIMITED_SUPPORT
-    if (self.shouldManageNetworkActivityIndicator) {
+    if (self.showNetworkActivityIndicator) {
```

Comment:
Was using the same variable that remained on Mixpanel.h and was part of the public API. I have no preference on the naming.